tensorflow_io/core/plugins/file_system_plugins.h

```diff
@@ -13,8 +13,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-#ifndef TENSORFLOW_IO_CORE_KERNELS_FILE_SYSTEM_PLUGINS_H
-#define TENSORFLOW_IO_CORE_KERNELS_FILE_SYSTEM_PLUGINS_H
+#ifndef TENSORFLOW_IO_CORE_PLUGINS_FILE_SYSTEM_PLUGINS_H
```

Comment:
Since we are here, maybe we can also update the very last line of the file as well?
```
#endif  // TENSORFLOW_IO_CORE_KERNELS_FILE_SYSTEM_PLUGINS_H
```

Reply:
@yongtang done !